src/utilities/__tests__/astFromValue-test.ts

```diff
@@ -69,6 +74,11 @@ describe('astFromValue', () => {
       value: '10000',
     });
 
+    expect(astFromValue(1n, GraphQLInt)).to.deep.equal({
```

Comment:
we should add a test case for a throw where we give a `bigint` value and it isn't a legal `GraphQLInt`

Reply:
Yes you're right! I added more! What do you think?